Build error if defined _WIN32_WINNT

Commit: 40a9a1060069d37d5f66912c6ee4cf165144774b

CMake:
```
cmake -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_STANDARD=17 -DCMAKE_CXX_FLAGS_INIT="/D_WIN32_WINNT=0x0601" -G "NMake Makefiles" ..\..
```

Error:
```
[ 87%] Building CXX object test/CMakeFiles/_test_malloc_lib_unload.dir/tbbmalloc
/test_malloc_lib_unload.cpp.obj
        cd F:\tmp\tbb\oneTBB-40a9a1060069d37d5f66912c6ee4cf165144774b\build\1921
x64\test
        "C:\Program Files\CMake\bin\cmake.exe" -E cmake_cl_compile_depends --dep
-file=CMakeFiles\_test_malloc_lib_unload.dir\tbbmalloc\test_malloc_lib_unload.cp
p.obj.d --working-dir=F:\tmp\tbb\oneTBB-40a9a1060069d37d5f66912c6ee4cf165144774b
\build\1921x64\test --filter-prefix="Note: including file: " -- C:\PROGRA~2\MICR
OS~1\2019\COMMUN~1\VC\Tools\MSVC\1421~1.277\bin\HostX64\x64\cl.exe @C:\Users\php
rus\AppData\Local\Temp\nm4DAC.tmp
Microsoft (R) C/C++ Optimizing Compiler Version 19.21.27702.2 for x64
Copyright (C) Microsoft Corporation.  All rights reserved.

cl   /TP -D_test_malloc_lib_unload_EXPORTS -IF:\tmp\tbb\oneTBB-40a9a1060069d37d5
f66912c6ee4cf165144774b\test\..\include -IF:\tmp\tbb\oneTBB-40a9a1060069d37d5f66
912c6ee4cf165144774b\test\.. -IF:\tmp\tbb\oneTBB-40a9a1060069d37d5f66912c6ee4cf1
65144774b\test /D_WIN32_WINNT=0x0601 /DWIN32 /D_WINDOWS  /GR /EHsc /O2 /Ob2 /DND
EBUG -MD -D_CRT_SECURE_NO_WARNINGS /GS /volatile:iso /FS -D_USRDLL -std:c++17 /s
howIncludes /FoCMakeFiles\_test_malloc_lib_unload.dir\tbbmalloc\test_malloc_lib_
unload.cpp.obj /FdCMakeFiles\_test_malloc_lib_unload.dir\ /FS -c F:\tmp\tbb\oneT
BB-40a9a1060069d37d5f66912c6ee4cf165144774b\test\tbbmalloc\test_malloc_lib_unloa
d.cpp

test_malloc_lib_unload.cpp
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(91): er
ror C3646: 'ModuleName': unknown override specifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(91): er
ror C4430: missing type specifier - int assumed. Note: C++ does not support defa
ult-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(92): er
ror C3646: 'hFile': unknown override specifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(92): er
ror C4430: missing type specifier - int assumed. Note: C++ does not support defa
ult-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(93): er
ror C3646: 'MappedAddress': unknown override specifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(93): er
ror C4430: missing type specifier - int assumed. Note: C++ does not support defa
ult-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(95): er
ror C3646: 'FileHeader': unknown override specifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(95): er
ror C4430: missing type specifier - int assumed. Note: C++ does not support defa
ult-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(99): er
ror C3646: 'LastRvaSection': unknown override specifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(99): er
ror C4430: missing type specifier - int assumed. Note: C++ does not support defa
ult-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(100): e
rror C3646: 'NumberOfSections': unknown override specifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(100): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(101): e
rror C3646: 'Sections': unknown override specifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(101): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(102): e
rror C3646: 'Characteristics': unknown override specifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(102): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(103): e
rror C3646: 'fSystemImage': unknown override specifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(103): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(104): e
rror C3646: 'fDOSImage': unknown override specifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(104): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(105): e
rror C3646: 'fReadOnly': unknown override specifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(105): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(106): e
rror C3646: 'Version': unknown override specifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(106): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(107): e
rror C3646: 'Links': unknown override specifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(107): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(108): e
rror C3646: 'SizeOfImage': unknown override specifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(108): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(135): e
rror C2065: 'CALLBACK': undeclared identifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(135): e
rror C2065: 'PFIND_DEBUG_FILE_CALLBACK': undeclared identifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(135): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(135): e
rror C2513: 'int': no variable declared before '='
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(135): e
rror C2143: syntax error: missing ';' before '('
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(136): e
rror C2146: syntax error: missing ')' before identifier 'FileHandle'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(136): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(136): e
rror C2146: syntax error: missing ';' before identifier 'FileHandle'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(139): e
rror C2059: syntax error: ')'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(142): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(142): e
rror C2086: 'int HANDLE': redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(136): n
ote: see declaration of 'HANDLE'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(142): e
rror C2146: syntax error: missing ';' before identifier 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(152): e
rror C2143: syntax error: missing ')' before '*'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(152): e
rror C2143: syntax error: missing ';' before '*'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(152): e
rror C2059: syntax error: ')'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(153): e
rror C2146: syntax error: missing ')' before identifier 'FileHandle'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(159): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(159): e
rror C2086: 'int HANDLE': redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(136): n
ote: see declaration of 'HANDLE'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(159): e
rror C2146: syntax error: missing ';' before identifier 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(169): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(169): e
rror C2086: 'int HANDLE': redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(136): n
ote: see declaration of 'HANDLE'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(169): e
rror C2146: syntax error: missing ';' before identifier 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(177): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(177): e
rror C2086: 'int HANDLE': redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(136): n
ote: see declaration of 'HANDLE'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(177): e
rror C2146: syntax error: missing ';' before identifier 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(187): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(187): e
rror C2086: 'int HANDLE': redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(136): n
ote: see declaration of 'HANDLE'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(187): e
rror C2146: syntax error: missing ';' before identifier 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(197): e
rror C2143: syntax error: missing ')' before '*'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(197): e
rror C2143: syntax error: missing ';' before '*'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(197): e
rror C2059: syntax error: ')'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(198): e
rror C2065: 'PCSTR': undeclared identifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(198): e
rror C2146: syntax error: missing ')' before identifier 'filename'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(203): e
rror C2143: syntax error: missing ';' before '__cdecl'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(205): e
rror C2146: syntax error: missing ')' before identifier 'hprocess'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(218): e
rror C2143: syntax error: missing ')' before '*'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(218): e
rror C2143: syntax error: missing ';' before '*'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(218): e
rror C2059: syntax error: ')'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(219): e
rror C2065: 'PCWSTR': undeclared identifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(219): e
rror C2146: syntax error: missing ')' before identifier 'filename'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(224): e
rror C2143: syntax error: missing ';' before '__cdecl'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(224): e
rror C2086: 'BOOL DECLSPEC_IMPORT': redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(203): n
ote: see declaration of 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(226): e
rror C2146: syntax error: missing ')' before identifier 'hprocess'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(239): e
rror C2143: syntax error: missing ')' before '*'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(239): e
rror C2143: syntax error: missing ';' before '*'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(239): e
rror C2059: syntax error: ')'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(240): e
rror C2146: syntax error: missing ')' before identifier 'FileHandle'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(246): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(246): e
rror C2086: 'int HANDLE': redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(136): n
ote: see declaration of 'HANDLE'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(246): e
rror C2146: syntax error: missing ';' before identifier 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(256): e
rror C2143: syntax error: missing ')' before '*'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(256): e
rror C2143: syntax error: missing ';' before '*'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(256): e
rror C2059: syntax error: ')'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(257): e
rror C2146: syntax error: missing ')' before identifier 'FileHandle'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(263): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(263): e
rror C2086: 'int HANDLE': redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(136): n
ote: see declaration of 'HANDLE'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(263): e
rror C2146: syntax error: missing ';' before identifier 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(273): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(273): e
rror C2086: 'int HANDLE': redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(136): n
ote: see declaration of 'HANDLE'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(273): e
rror C2146: syntax error: missing ';' before identifier 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(281): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(281): e
rror C2086: 'int HANDLE': redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(136): n
ote: see declaration of 'HANDLE'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(281): e
rror C2146: syntax error: missing ';' before identifier 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(291): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(291): e
rror C2086: 'int HANDLE': redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(136): n
ote: see declaration of 'HANDLE'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(291): e
rror C2146: syntax error: missing ';' before identifier 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(301): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(301): e
rror C2146: syntax error: missing ';' before identifier 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(307): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(307): e
rror C2146: syntax error: missing ';' before identifier 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(317): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(317): e
rror C2086: 'int PVOID': redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(306): n
ote: see declaration of 'PVOID'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(317): e
rror C2146: syntax error: missing ';' before identifier 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(326): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(326): e
rror C2146: syntax error: missing ';' before identifier 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(334): e
rror C4430: missing type specifier - int assumed. Note: C++ does not support def
ault-int
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(334): e
rror C2086: 'int PVOID': redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(306): n
ote: see declaration of 'PVOID'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(334): e
rror C2146: syntax error: missing ';' before identifier 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(410): e
rror C2143: syntax error: missing ';' before '__cdecl'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(410): e
rror C2086: 'BOOL DECLSPEC_IMPORT': redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(203): n
ote: see declaration of 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(412): e
rror C2065: 'PCSTR': undeclared identifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(412): e
rror C2146: syntax error: missing ')' before identifier 'RootPath'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(418): e
rror C2143: syntax error: missing ';' before '__cdecl'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(418): e
rror C2086: 'BOOL DECLSPEC_IMPORT': redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(203): n
ote: see declaration of 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(420): e
rror C2065: 'PCWSTR': undeclared identifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(420): e
rror C2146: syntax error: missing ')' before identifier 'RootPath'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(426): e
rror C2143: syntax error: missing ')' before '*'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(426): e
rror C2143: syntax error: missing ';' before '*'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(426): e
rror C2059: syntax error: ')'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(427): e
rror C2065: 'PCSTR': undeclared identifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(427): e
rror C2146: syntax error: missing ')' before identifier 'FilePath'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(432): e
rror C2143: syntax error: missing ';' before '__cdecl'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(432): e
rror C2086: 'BOOL DECLSPEC_IMPORT': redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(203): n
ote: see declaration of 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(434): e
rror C2146: syntax error: missing ')' before identifier 'hProcess'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(443): e
rror C2143: syntax error: missing ')' before '*'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(443): e
rror C2143: syntax error: missing ';' before '*'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(443): e
rror C2059: syntax error: ')'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(444): e
rror C2065: 'PCWSTR': undeclared identifier
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(444): e
rror C2146: syntax error: missing ')' before identifier 'FilePath'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(449): e
rror C2143: syntax error: missing ';' before '__cdecl'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(449): e
rror C2086: 'BOOL DECLSPEC_IMPORT': redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(203): n
ote: see declaration of 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(451): e
rror C2146: syntax error: missing ')' before identifier 'hProcess'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(460): e
rror C2143: syntax error: missing ';' before '__cdecl'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(460): e
rror C2086: 'BOOL DECLSPEC_IMPORT': redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(203): n
ote: see declaration of 'DECLSPEC_IMPORT'
C:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um\dbghelp.h(460): f
atal error C1003: error count exceeds 100; stopping compilation
NMAKE : fatal error U1077: '"C:\Program Files\CMake\bin\cmake.exe"' : return cod
e '0x2'
Stop.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio\2019
\Community\VC\Tools\MSVC\14.21.27702\bin\HostX64\x64\nmake.exe"' : return code '
0x2'
Stop.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio\2019
\Community\VC\Tools\MSVC\14.21.27702\bin\HostX64\x64\nmake.exe"' : return code '
0x2'
Stop.
```

Solution:

Add
```c++
        #define NOMINMAX
        #include <windows.h>
```
before
 https://github.com/oneapi-src/oneTBB/blob/40a9a1060069d37d5f66912c6ee4cf165144774b/test/common/utils_report.h#L56

--- a/test/common/utils_report.h
+++ b/test/common/utils_report.h
@@ -53,6 +53,10 @@
         // Suppress "typedef ignored ... when no variable is declared" warning by vc14
         #pragma warning (push)
         #pragma warning (disable: 4091)
+        #ifndef NOMINMAX
+            #define NOMINMAX
+        #endif
+        #include <windows.h>
         #include <dbghelp.h>
         #pragma warning (pop)
         #pragma comment (lib, "dbghelp.lib")

